app/Console/Commands/Migrations/MigrateContacts.php

```diff
@@ -0,0 +1,127 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Console\Commands\Migrations;
+
+use App\Domain\Auth\Models\UserModel;
+use App\Domain\Contacts\Contact;
+use App\Domain\Contacts\Repositories\ContactRepository;
+use App\Insightly\InsightlyMapping;
+use App\Insightly\Repositories\InsightlyMappingRepository;
+use App\Insightly\Resources\ResourceType;
+use Exception;
+use Illuminate\Console\Command;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\ModelNotFoundException;
+use Illuminate\Database\UniqueConstraintViolationException;
+use Ramsey\Uuid\Uuid;
+use Ramsey\Uuid\UuidInterface;
+use Spatie\Activitylog\Facades\CauserResolver;
+
+final class MigrateContacts extends Command
+{
+    use ReadCsvFile;
+
+    protected $signature = 'migrate:contacts';
+
+    protected $description = 'Migrate contacts provided in the contacts.csv file (database/project-aanvraag/contacts.csv)';
+
+    public function __construct(
+        private readonly ContactRepository $contactRepository,
+        private readonly InsightlyMappingRepository $insightlyMappingRepository
+    ) {
+        parent::__construct();
+    }
+
+    public function handle(): int
+    {
+        Model::unsetEventDispatcher();
+
+        CauserResolver::setCauser(UserModel::createSystemUser());
+
+        $this->info('Migrating contacts...');
+
+        $contacts = $this->readCsvFile('database/project-aanvraag/contacts.csv');
+        $migrationContacts = array_map(fn (array $contact) => new MigrationContact($contact), $contacts);
+        $contactsCount = count($migrationContacts);
+
+        if ($contactsCount <= 0) {
+            $this->warn('No contacts to import');
+            return 0;
+        }
+
+        if (!$this->confirm('Are you sure you want to import ' . $contactsCount . ' contacts?')) {
+            return 0;
+        }
+
+        foreach ($migrationContacts as $migrationContact) {
+            $contactId = Uuid::uuid4();
+
+            $this->info($contactId . ' - Started importing contact ' . $migrationContact->email());
+
+            try {
+                $this->migrateContact($contactId, $migrationContact);
+            } catch (UniqueConstraintViolationException) {
+                $this->warn($contactId . ' - Contact ' . $migrationContact->email() . ' already exists');
+            } catch (Exception $e) {
+                $this->error($contactId . ' - Failed importing contact ' . $migrationContact->email() . ': ' . $e->getMessage());
+            }
+
+            $this->info($contactId . ' - Ended importing contact ' . $migrationContact->email());
+            $this->info('---');
+        }
+
+        return 1;
```

Comment:
Why is an error code (not 0) returned when the contacts have been migrated?
Or is it 0 (false) and 1 (true)?

Reply:
This is indeed wrong, because `0` means `success`